packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -118,9 +118,17 @@ def test_pod_logs(dcos_api_session):
 
     with dcos_api_session.marathon.deploy_pod_and_cleanup(pod_definition):
         url = get_task_url(dcos_api_session, pod_id)
+        url_parts = url.split('/')
```

Comment:
Small nit, but if all you are using `url_parts` for is to get the `container_id`, why don't you just move your comment below up here and make this line `container_id = url.split('/')[-1]`